infra/src/cli/render-secrets.ts

```diff
@@ -20,41 +25,44 @@ export const renderSecrets = async (service: string) => {
     (chart) => generateYamlForEnv(uberChart, ...chart.dev).services,
```

Comment:
I'm not sure at what point this got changed ... but previously you could request environment variables for the whole project without specifying which part, e.g.
```
yarn get-secrets air-discount-scheme
```
used to work ... but now you've to request
```
yarn get-secrets air-discount-scheme-web
yarn get-secrets air-discount-scheme-backend
yarn get-secrets air-discount-scheme-api
```

Reply:
previously it only used SSM prefixes, e.g. if you did `yarn get-secrets air-discount-scheme` it would only fetch ssm parameters with prefix `/k8s/air-discount-scheme*`. one of the problems with this was that a lot of projects share secrets (examples include `api`s usage of `/k8s/shared/api/RSK_API_USERNAME` and `/k8s/documents/POSTHOLF_CLIENTID`

Reply:
Now it uses the actual services defined in the dsl, so you get exactly what you need, nothing more, nothing less

Reply:
I mean we have the dependencies of the services so we can pull all the secrets for all of them but would you use them in the same shell?

Reply:
Nevermind, it is not in the shell but a file. But my point still stands about having the option for the related services.